diff --git a/pri/quazipdetect.pri b/pri/quazipdetect.pri
index 07bd5621..ab996a46 100644
--- a/pri/quazipdetect.pri
+++ b/pri/quazipdetect.pri
@@ -2,15 +2,15 @@
 
 message("Using Fritzing quazip detect script.")
 
-QUAZIP_VERSION=1.4
-QUAZIP_PATH=$$absolute_path($$PWD/../../quazip-$$QT_VERSION-$$QUAZIP_VERSION)
+QUAZIP_VERSION=1.5
+QUAZIP_PATH=/app
 QUAZIP_INCLUDE_PATH=$$QUAZIP_PATH/include/QuaZip-Qt6-$$QUAZIP_VERSION
 QUAZIP_LIB_PATH=$$QUAZIP_PATH/lib
 
 SOURCES += \
 	src/zlibdummy.c \
 
-exists($$QUAZIP_PATH) {
+exists($$QUAZIP_INCLUDE_PATH) {
 		message("found quazip in $${QUAZIP_PATH}")
 	} else {
 		error("quazip could not be found at $$QUAZIP_PATH")
