diff --git a/pri/clipper1detect.pri b/pri/clipper1detect.pri
index 5c6505d2..465bb5b1 100644
--- a/pri/clipper1detect.pri
+++ b/pri/clipper1detect.pri
@@ -5,8 +5,8 @@ message("Using fritzing Clipper 1 detect script.")
 unix {
     message("including Clipper1 library on linux or mac")
 
-    exists($$absolute_path($$PWD/../../Clipper1)) {
-	            CLIPPER1 = $$absolute_path($$PWD/../../Clipper1/6.4.2)
+    CLIPPER1 = /app
+    exists($$absolute_path($${Clipper1})) {
 				message("found Clipper1 in $${CLIPPER1}")
 			}
 }
