diff --git a/pri/spicedetect.pri b/pri/spicedetect.pri
index 76795bd8..555258d5 100644
--- a/pri/spicedetect.pri
+++ b/pri/spicedetect.pri
@@ -15,8 +15,7 @@
 
 
 message("Using fritzing ngspice detect script.")
-NGSPICEPATH = ../../ngspice-42
-NGSPICEPATH = $$absolute_path($${NGSPICEPATH})
+NGSPICEPATH = /app
 
 exists($$NGSPICEPATH) {
 	message("found $${NGSPICEPATH}")
@@ -24,3 +23,6 @@ exists($$NGSPICEPATH) {
 } else {
     error("ngspice not found in $${NGSPICEPATH}")
 }
+
+LIBS += -L/app/lib/ngspice -lngspice
+QMAKE_RPATHDIR += /app/lib/ngspice
